users/views.py

```
@@ -99,21 +99,18 @@ def registerPage(request):
-            else:
-                profile.is_verified = True
-                profile.save()
-                return redirect('verify-email')
-        except Exception as e:
-            messages.error(request, 'Invalid verification link.')
-            return redirect('register')
-    else:
-        return render(request, 'users/auth-process.html', {'page': 'verify-email'})
+    try:
+        profile = get_object_or_404(Profile, verification_code=code)
+        if profile.is_verified == True:
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if profile.is_verified:` for truth checks

Replace with `profile.is_verified`

(E712)